drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -339,9 +390,23 @@ local function info_changed(driver, device, event, args)
   end
 end
 
+local function device_added(driver, device, event, args)
+  local limit_read = im.InteractionRequest(im.InteractionRequest.RequestType.READ, {})
```

Comment:
We really only need to do once per device. What about checking the currently capability status and only doing the reads if the attribute hasn't been set yet?

Reply:
Are we purposefully not doing this for existing devices? I would think we'd want to do it for all devices once

Reply:
My intention with my code changes was that the code for the behavior we want to see going forward would be put in first, and later there would be some additional migration code added that's only live for a week or two before being removed.

Migration code like moving off of old profiles, querying for the bounds, etc.

Reply:
This was moved to `init` in b76b11afefc0552a0e3cf4329c6381d570277552 so that it will work for existing devices as well as new devices.

Reply:
Also, the BOUNDS_CHECKED flag was introduced to ensure that it will only occur once per device.